src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java

```diff
@@ -367,7 +367,9 @@ private void onPluginStart() {
         // Armor Update Task
         if (config.getBoolean("options.enable-armor-effects")) {
             new SlimefunArmorTask().schedule(this, config.getInt("options.armor-update-interval") * 20L);
-            new RadiationTask().schedule(this, config.getInt("options.radiation-update-interval") * 20L);
+            if (config.getBoolean("options.enable-radiation")) {
+                new RadiationTask().schedule(this, config.getInt("options.radiation-update-interval") * 20L);
+            }
```

Comment:
Should this be moved outside of the enable-armor-effects condition?

Reply:
The reason this was inside the enable-armor-effects condition is probably that the hazmat suit wouldn't work if armor effects were disabled, making it impossible to handle radioactive items if the radiation task wouldn't also be disabled.

Reply:
Therefore `options.enable-armor-effects` = false and `options.enable-radiation` = true is an invalid state.
I would strongly suggest to add a console warning in this case and warn that radiation will not work unless armor effects are enabled.

Reply:
Added a warning for the mentioned state